src/gui/DatabaseOpenWidget.cpp

```diff
@@ -46,18 +46,12 @@ DatabaseOpenWidget::DatabaseOpenWidget(QWidget* parent)
 
     m_ui->messageWidget->setHidden(true);
 
-    QFont font = m_ui->labelHeadline->font();
-    font.setBold(true);
-    font.setPointSize(font.pointSize() + 2);
-    m_ui->labelHeadline->setFont(font);
+    m_ui->labelHeadline->setText(tr("Unlock KeePassXC Database"));
```

Comment:
I find the font to be rather small and uninviting. We should place the purpose of the dialog (this text) and the database we are unlocking in a prominent and dedicated position. Here is a small mockup:

![image](https://user-images.githubusercontent.com/2809491/59726358-43b2f380-91ff-11e9-9ada-6b3976e5609d.png)

Reply:
You can wrap the text with ```<h1>``` tags to make it bigger, but it still seems small to me on Windows 10.

Reply:
On Windows everything looks small. I think we can put it above and make it larger, but this looks a bit unbalanced to me. Need to think about how we best solve this.

Reply:
Oh right, well then at least the file path would be nice.

I like the way OKThanks mocked up our "onboarding" process. This is kind of the look I was going for by titling the widget:

![image](https://user-images.githubusercontent.com/2809491/59765978-37fe1600-926d-11e9-96e7-1decf6c7ce3e.png)

Reply:
I would like to give KeePassXC a more modern look (perhaps not as "web-appy" or "bootstrappy" as the screenshot above, but similar), but it's hard to do with Qt if we want to go for a platform-native look. I think the only way here would be QtQuick with custom styling. But I fear we would alienate a lot of users with an aggressive redesign of everything.

Reply:
There is a middle ground here. We are currently on the "nothing custom, nothing special, very technical" side of the graph.

Reply:
I know. I would love to do a bit more in the "styled" direction, but it's hard to do with QWidgets.